src/pages/HomePage.java

```diff
@@ -0,0 +1,32 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.How;
+import org.openqa.selenium.support.PageFactory;
+
+public class HomePage extends BasePage{
+
+    BasePage page = new BasePage(driver);
```

Comment:
Please remove this variable, because you extend this class.

Reply:
@vladimirKorzkuhov
I am not sure if you read my notes. But I am getting test failures if I declare it somewhere else or remove it.